UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamageDiscriptors.cs

```diff
@@ -23,88 +25,117 @@ public partial class BodyPart
 		private string pireceDamageDescOnLARGE= "{readableName} suffers from a Ruptured Cavity.";
 		[SerializeField] private string internalDamageDesc	 = "This {readableName} is suffering from internal damage.";
 		[SerializeField] private string externalBleedingDesc = "This {readableName} is bleeding due to an open wound.";
+		[SerializeField, NaughtyAttributes.EnableIf("CanBeBroken")] private string BoneFracturedLvlThreeDesc = "This {readableName} is suffering Hairline Fracture.";
```

Comment:
```suggestion
		[SerializeField, NaughtyAttributes.EnableIf(nameof(CanBeBroken))] private string BoneFracturedLvlThreeDesc = "This {readableName} is suffering Hairline Fracture.";
```

Use ``nameof`` when referencing a variable like this, so if in the future the variable name changes the reference won't break.

Reply:
done 👍